Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2332,14 +2334,15 @@ public void applyReferralCode(final String code, final BranchReferralInitListene
      * user selected clients.</p>
      *
      * @param builder A {@link io.branch.referral.Branch.ShareLinkBuilder} instance to build share link.
+     * @return The instance of the {@link Dialog} holding the share view.
      */
-    private void shareLink(ShareLinkBuilder builder) {
+    private Dialog shareLink(ShareLinkBuilder builder) {
```

Comment:
Unlink the `create` method in `alertBuilder` we are using `shareLink()` method to start sharing.That method already say it will create  chooser dialog for them. So I guess for the newer users we can survive with this. Also if user explicitly call `Show() ` when the dialog is up , there is nothing change